stories/Atom/Images/Imagecredit/imagecredit.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/breakpoints";
+
+/* credit image*/
+.credit--image-author {
+    margin-bottom: 10px;
+    span,

```

Comment:
can we use the classes on the span and then provide the styling and try to avoid the nesting